products/ghc.md

```diff
@@ -0,0 +1,122 @@
+---
+title: GHC
+iconSlug: haskell
+category: lang
+tags: haskell-runtime
+permalink: /ghc
+versionCommand: ghc --version
+releasePolicyLink: https://gitlab.haskell.org/ghc/ghc/-/wikis/GHC-status
+changelogTemplate: https://downloads.haskell.org/ghc/__LATEST__/docs/users_guide/__LATEST__-notes.html
+
+eoasColumn: Further releases planned
+eolColumn: Recommended for use
```

Comment:
These two are very vague. We need something that aligns with the verbs that we use in the table ("ends in {duration}", "ended {duration} ago").

```suggestion
eoasColumn: Further releases planned
eolColumn: Recommended for use
```

We can give them better names and then explain the meanings in the description text.

Reply:
Vague indeed, but I'll ask a suggestion here :thinking:

First, I didn't want to depart too far from the [source table](https://gitlab.haskell.org/ghc/ghc/-/wikis/GHC-status).

Then, to clarify my affiliation: I landed one small bugfix (and still waiting for it to get released), but overall the GHC team would be perfectly justified in calling me an outside contributor.

Then, I make sense of the two columns like so:
* **Further releases planned** — "Yes" means there're known work items (patches) to be batched in to the next minor release. "No" in this column doesn't mean EOL; it simply means there're no patches making their way into the next minor release. For whatever reasons.
* **Recommended for use** — "No" here essentially signals EOL: the version might be perfectly usable anyway, but bug-reports (if any) will be asked to retest on a newer version, and patches will not be considered.

Reply:
... I also refreshed the release data. Perhaps this now gives a better idea about the former column:

![image](https://github.com/user-attachments/assets/0e82225f-96b0-46fa-b5dc-f8a902b789ac)

i.e. the **Further releases planned** — is merely a reflection of milestones where the project is moving toward. It's useful to see IMO, because these releases often take months to come out.